components/plan-build/bin/shared.bash

```diff
@@ -86,28 +88,34 @@ _render_metadata_FILES() {
   pushd "$CACHE_PATH" > /dev/null
   build_line "Generating blake2b hashes of all files in the package"
 
-  find $pkg_prefix -type f \
+  # shellcheck disable=2154
+  find "$pkg_prefix" -type f \
     | sort \
-    | hab pkg hash > ${pkg_name}_blake2bsums
+    | hab pkg hash > "${pkg_name}"_blake2bsums
 
   build_line "Generating signed metadata FILES"
-  $HAB_BIN pkg sign --origin $pkg_origin ${pkg_name}_blake2bsums $pkg_prefix/FILES
+  # shellcheck disable=2154
+  $HAB_BIN pkg sign --origin "$pkg_origin" "${pkg_name}"_blake2bsums "$pkg_prefix"/FILES
   popd > /dev/null
 }
 
 _render_metadata_IDENT() {
   debug "Rendering IDENT metadata file"
-  echo "${pkg_origin}/${pkg_name}/${pkg_version}/${pkg_release}" >> $pkg_prefix/IDENT
+  # shellcheck disable=2154
+  echo "${pkg_origin}/${pkg_name}/${pkg_version}/${pkg_release}" >> "$pkg_prefix"/IDENT
 }
 
 _render_metadata_INTERPRETERS() {
     local metadata_file_name="INTERPRETERS"
 
+    # shellcheck disable=2154
     if [[ ${#pkg_interpreters[@]} -gt 0 ]]; then
         debug "Rendering ${metadata_file_name} metadata file"
-        local interpreters="$(printf "${pkg_prefix}/%s\n" ${pkg_interpreters[@]})"
-        printf "%s\n" ${pkg_interpreters[@]} \
-            | sed "s|^|${pkg_prefix}/|" > $pkg_prefix/${metadata_file_name}
+        local interpreters
+        # shellcheck disable=2034
```

Comment:
```
In ./components/plan-build/bin/shared.bash line 115:
        interpreters="$(printf "${pkg_prefix}/%s\n" "${pkg_interpreters[@]}")"
        ^-- SC2034: interpreters appears unused. Verify it or export it.
```
This `interpreters` local variable doesn't seem to be used for anything. I propose removing it.